### PR TITLE
Add Plone to Rancher Community Catalog

### PR DESCRIPTION
**Pull request by avoinea**
Now that we have an official Docker image for Plone, we should also have it within Rancher catalog ;)

### CODE REVIEW

**Comment by galal-hussein** [comment]
@avoinea Thanks for the PR.

**Comment by galal-hussein** [comment]
LGTM.

File: templates/plone/config.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+name: "Plone"
+description: |
+  Plone is the premier Python-based CMS
+version: "5.0-rancher1"
+maintainer: "Alin Voinea <alin.voinea@gmail.com>"
+minimum_rancher_version: v1.0
+category: "Content Management Systems"
```
Comment (avoinea) [inline templates/plone/config.yml]: yap

**Comment by galal-hussein** [comment]
@avoinea Thanks.

**Pull request merged**